### DISCUSSION TITLE
Brødsmulesti

### DISCUSSION THREAD

**Discussion by wkillerud** [fremtind/jokul#2716] [Annet]
BM har laget en variant, men prinsippet er noe flere kan få bruk for. Lager en diskusjon her for å ha et sted å logge litt informasjon.

**Comment by kristineSorensen** [comment]
Vi har lekt oss litt med BMs brødsmulesti i noen skisser på seOpp - godt mulig dette kan være aktuelt for oss også.

**Comment by wkillerud** [comment]
Jeg tok en titt på koden til BM og tilpasset den litt i #2722 

Forhåndsvisning her: https://jokul.fremtind.no/preview/feat/brodsmulesti/komponenter/breadcrumb

Ta gjerne en titt på hva jeg har skrevet om komponenten og bruken av den, og se om dere er enige.

**Comment by AudunBerdal** [comment]
Synes det blir rart å bruke et anførselstegn (») som pil i dette tilfellet. Hadde det ikke vært mer riktig å bruke en vanlig pil her slik vi gjør med navigasjonslenker? Brødsmulestien er jo i prinsippet en navigasjonslenke også. 

Se eksempel på fremtind.no

**Comment by ivnai** [comment]
Enig med Audun at "»" er rart. Men tror at den pilen vi bruker nå er for kompleks å gjenta flere ganger i en linje og i tillegg for kompleks for en liten size som vi trenger her. Ville vurdere slash eller mikro-chevron. Bruk av chevron trekker kanskje ut igjen diskusjonen om piler vs chevrons.

**Comment by onordbo** [comment]
Litt enig og uenig i pil-debatten over. Sånn rent bibliotek og brandmessig så vil jeg si det ligger utenfor formspråket vårt å bruke anførselstegn som et "videre" ikon eller en indikator. Men pil-ikonene vi har tilgjengelige fra før av kan være som @ivnai nevner for komplekse, forvirrende og gjøre det litt trangt/støyete. Jeg er usikker på om den resonnerer nok visuelt sett med sånn brukere vanligvis opplever mønsteret. Uten at jeg graver opp for mange eksempler fra internett, så er mitt inntrykk at det er ganske vanlig med en chevron eller skråstrek, men vi har hverken den ene eller den andre i biblioteket vårt.

Prøvde meg kjapt på et par varianter, 8pt og 12pt avstand, og en med "fadet" pil for litt mindre visuelt fotavtrykk, men jeg er ikke så sikker på om det er veien å gå.

## Hvit
![image](https://user-images.githubusercontent.com/4263132/153418912-5afd059a-3687-47d8-95d5-89a099443f25.png)

## Sand
![image](https://user-images.githubusercontent.com/4263132/153418955-bed4e14d-e302-43c7-8fd7-ab606fd3493a.png)

## Dark mode
![image](https://user-images.githubusercontent.com/4263132/153419666-01ec7df3-036c-45b2-bac8-63d6092525c2.png)

Ta en kikk på @wkillerud's [forslag i biblioteket](https://www.figma.com/file/jjMeSoNPQJgTUIOyG3jGsK/?node-id=4446%3A43576), og lag gjerne en remix eller kom med andre forslag @ivnai og @AudunBerdal 😄

**Comment by Murstam** [comment]
Snakket om det på VD-forum i dag. Vi bruker pil i forbindelse med navigasjon og åpne/lukke, mens her fungerer symbolet mer som et skilletegn. Vi ønsker derfor ikke å bruke pil, men går heller for chevron for å tone ned uttrykket. Lou oppdaterer branch. 

<img width="920" alt="Skjermbilde 2022-02-11 kl  10 24 27" src="https://user-images.githubusercontent.com/47025023/153567335-9e5d1322-9285-45eb-9fe8-d8ebba1b545a.png">

**Comment by wkillerud** [comment]
Da er versjon 1 av Breadcrumb ute både i Figma og [på npm](https://www.npmjs.com/package/@fremtind/jkl-breadcrumb-react) 🎉

**Reply by joms** [reply]
![](https://media0.giphy.com/media/Y0sw2N6ySq5kEwDO7S/giphy.gif?cid=ecf05e4767j0rc57tt33e88p2nx3ia6gx5lcflnz1wb0452u&rid=giphy.gif&ct=g)
